contrib/build_sdk.sh

```diff
@@ -477,14 +477,15 @@ libuv_pkg() {
 
     package_extract $name $libuv_file $libuv_dir
 
+    export OLD_CFLAGS="$CFLAGS"
```

Comment:
why export?

Reply:
thanks - I've created a much cleaner version of this PR and included the fix for this.